recipes/igraph/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "igraph" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://igraph.org/nightly/get/c/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }} 
+
+build:
+  skip: True  # [ win ]
+  number: 3
+
+requirements:
+  build:
+    - toolchain
+    - gmp
+    - libxml2
+    - pkg-config
+  run:
+    - gmp 
+    - libxml2
+    - pkg-config
```

Comment:
You are right. If the person using the library needs it, it would be at their own build-time. I am removing the line.